### PR TITLE
ft - Select Movers

### PR DESCRIPTION
**Pull request by Nicanor008**
#### What does this PR do?
- Allow users to select available movers

#### Description of Task to be completed?
- [x] Search ui for a mover
- [x] Movers displayed on a dropdown select
- [x] store(actions and reducers) to sent selected movers to the API 

#### Background Context
- None

#### Screenshot
Screenshot 2019-10-02 at 15 25 32

### CODE REVIEW

File: src/components/MenuLayout.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ const MenuLayout = ({
                     <Dropdown.Item as={Link} to="/create">
                       New
                     </Dropdown.Item>
-                    <Dropdown.Item>My Orders</Dropdown.Item>
+                    <Dropdown.Item as={Link} to="/select-movers">My Orders</Dropdown.Item>
```
Comment (joyjoie) [inline src/components/MenuLayout.js]: indent line 51 to be uniform with the rest of the tags

File: src/containers/client/client.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+.hide {
+  display: none;
+}
+
+.emptyDropdownList {
+    background-color: rgb(198, 201, 202) !important;
+    color: black !important;
+}
+
+
```
Comment (joyjoie) [inline src/containers/client/client.scss]: remove line spaces

File: src/store/Constants/index.js [hunk 3]
```diff
@@ -29,3 +29,7 @@ export const CREATE_ORDER_ERROR = "CREATE_ORDER_ERROR";
 export const GET_ORDERS = "GET_ORDERS";
 export const GET_ORDERS_SUCCESS = "GET_ORDERS_SUCCESS";
 export const GET_ORDERS_ERROR = "GET_ORDERS_ERROR";
+
```
Comment (joyjoie) [inline src/store/Constants/index.js]: remove empty space

File: src/store/Order/actions.js [hunk 4]
```diff
@@ -29,22 +32,25 @@ export const orderFailure = (error, type) => ({
...
-        return toastr.success("Order Submitted");
+        toastr.success("Order Submitted");
+        window.location.href = `/select-movers/${response.data.id}`;
     }, 400);
   } catch (error) {
     dispatch(orderFailure({ error }, CREATE_ORDER_ERROR));
-    return toastr.error("Something Went wrong. Try again");
+    return toastr.error("Something Went wrong. Try again", "Oops, Failed");
   }
 };
```
Comment (joyjoie) [inline src/store/Order/actions.js]: remove space in line 50 and 51

**Comment by Nicanor008** [comment]
Closed because I have merged it already

**Pull request closed without merging**